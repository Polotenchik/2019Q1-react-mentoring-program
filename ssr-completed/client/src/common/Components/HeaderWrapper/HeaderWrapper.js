import styled from 'styled-components';

const HeaderWrapper = styled.div`
    width: 80%;
    margin: 0 auto;
    padding: 10px 0 20px;
`;

export default HeaderWrapper;
