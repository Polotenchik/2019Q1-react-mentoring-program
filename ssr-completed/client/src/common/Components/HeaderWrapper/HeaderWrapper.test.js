import React from 'react';
import { shallow } from 'enzyme';
import HeaderWrapper from './HeaderWrapper';

describe('HeaderWrapper Snapshot', () => {
    it('HeaderWrapper Component should render correctly with no props', () => {
        const component = shallow(<HeaderWrapper />);
        expect(component).toMatchSnapshot();
    });
});