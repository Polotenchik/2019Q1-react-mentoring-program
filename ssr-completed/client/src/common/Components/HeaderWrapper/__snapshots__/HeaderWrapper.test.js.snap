// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderWrapper Snapshot HeaderWrapper Component should render correctly with no props 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    width: 80%;
    margin: 0 auto;
    padding: 10px 0 20px;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
